hedera-node/hedera-mono-service/src/testFixtures/java/com/hedera/test/factories/scenarios/TokenKycRevokeScenarios.java

```diff
@@ -31,6 +32,12 @@ public PlatformTxnAccessor platformTxn() throws Throwable {
         }
     },
     REVOKE_WITH_MISSING_TOKEN {
+        @Override
+        public PlatformTxnAccessor platformTxn() throws Throwable {
+            return PlatformTxnAccessor.from(newSignedTokenCreate().missingAdmin().get());
+        }
+    },
```

Comment:
Why is this test scenario removed ? Is it not working with latest changes ?

Reply:
I believe it was just renamed to `REVOKE_WITH_INVALID_TOKEN` (line 40) to reflect the name more accurately, and a new scenario was added

`MISSING TOKEN` ==> no `tokenID` in the body
`INVALID TOKEN` ==> there is `tokenID` in the body, but not present in the state